### PR TITLE
Create and print pdf document (invoice for payment)

### PR DESCRIPTION
**Pull request by abarmina**
Could not convert the date in the document

### CODE REVIEW

File: src/services/http-client.js [hunk 1]
```diff
@@ -3,7 +3,7 @@ import store from '@/store'
 import tokenService from '@/services/token'
 
 const options = {
-  baseURL: process.env.VUE_APP_API_HOST,
+  baseURL: "https://pdrc-calculator-strapi.herokuapp.com",
```
Comment (klikkn) [inline src/services/http-client.js]: no no no)

File: package.json [hunk 2]
```diff
@@ -11,9 +11,11 @@
   "dependencies": {
     "axios": "^0.19.0",
     "core-js": "^3.3.2",
+    "date-fns": "^2.11.1",
```
Comment (klikkn) [inline package.json]: needless

File: src/containers/OrderUpdate.vue [hunk 3]
```diff
@@ -75,6 +79,90 @@ export default {
...
+                style: "subheader"
+              }
+            ],
+            style: "header"
+          },
+          {
+            text: [
+              `${this.$t("clientName")}: ${this.order.clientName}\n`,
+              `${this.$t("phoneNumber")}: ${this.order.phoneNumber}\n`,
+              `${this.$t("carModel")}: ${this.order.make}\n`,
```
Comment (klikkn) [inline src/containers/OrderUpdate.vue]: why?) make and model - different things, make it's make

File: src/i18n/ru.json [hunk 4]
```diff
@@ -84,13 +85,15 @@
     "square": "Выберите площадь",
     "year": "Выберите год"
   },
+  "score": "Счет на оптату",
```
Comment (klikkn) [inline src/i18n/ru.json]: invoice

File: package.json (hunk 2 above)
Comment (klikkn) [reply package.json]: you can use 

import dayjs from 'dayjs'
dayjs(value).format('DD.MM.YYYY')

File: src/containers/OrderUpdate.vue (hunk 3 above)
Comment (klikkn) [reply src/containers/OrderUpdate.vue]: maker maybe

File: src/i18n/ru.json (hunk 4 above)
Comment (klikkn) [reply src/i18n/ru.json]: на оплату

File: src/containers/OrderUpdate.vue [hunk 5]
```diff
@@ -75,6 +79,90 @@ export default {
...
+              `${this.$t("carNumber")}: ${this.order.carNumber}\n`
+            ],
+            style: "text"
+          },
+          {
+            table: {
+              widths: ["*", 125, 75, 75],
+              body: [
+                [
+                  { text: `${this.$t("part")}`, style: "tableHeader" },
```
Comment (klikkn) [inline src/containers/OrderUpdate.vue]: just
{ text: this.$t("part"), style: "tableHeader" }

File: src/services/http-client.js (hunk 1 above)
Comment (abarmina) [reply src/services/http-client.js]: we used this link because it dont`t work else. but it works now. wonders!

File: package.json (hunk 2 above)
Comment (abarmina) [reply package.json]: perfectly!
deleted "date-fns". changed date.

File: src/i18n/ru.json (hunk 4 above)
Comment (abarmina) [reply src/i18n/ru.json]: done

File: src/containers/OrderUpdate.vue (hunk 3 above)
Comment (abarmina) [reply src/containers/OrderUpdate.vue]: OK

File: src/containers/OrderUpdate.vue (hunk 5 above)
Comment (abarmina) [reply src/containers/OrderUpdate.vue]: agree

**Comment by abarmina** [comment]
All comments considered

File: src/containers/OrderUpdate.vue [hunk 6]
```diff
@@ -75,6 +80,90 @@ export default {
...
+
+      const { categories, squares, parts } = this.params;
+
+      var dd = {
+        content: [
+          {
+            stack: [
+              this.$t("invoice"),
+              {
+                text: `${dayjs(this.order.date).format("DD.MM.YYYY")}`,
```
Comment (klikkn) [inline src/containers/OrderUpdate.vue]: just dayjs(this.order.date).format("DD.MM.YYYY")

Comment (abarmina) [reply src/containers/OrderUpdate.vue]: done

**Pull request merged**